### PR TITLE
Added Tag component

### PR DESCRIPTION
**Pull request by ssb-cgn**

### CODE REVIEW

File: src/components/Tag/index.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Tag = ({
+	children, className, icon, onClick,
+}) => (
+	<span
```
Comment (kristofferlund) [inline src/components/Tag/index.jsx]: Span is a non interactive element and should not be used as a button unless there's a good reason for it. A button would be more fitting, and the "reset-button" mixin will remove the default styling of it.
https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-noninteractive-element-interactions.md

File: src/components/Tag/tag.story.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import centered from '@storybook/addon-centered/react';
+import { Star } from 'react-feather';
+import Tag from './index';
+
+storiesOf('Tag', module).addDecorator(centered)
+	.add('Tag', () => (
+		<div style={{ display: 'flex' }}>
```
Comment (kristofferlund) [inline src/components/Tag/tag.story.jsx]: Doesn't really do anything, but it doesn't ruin anything either

File: src/components/Tag/index.jsx (hunk 1 above)
Comment (ssb-cgn) [reply src/components/Tag/index.jsx]: Fixed

File: src/components/Tag/tag.story.jsx (hunk 2 above)
Comment (ssb-cgn) [reply src/components/Tag/tag.story.jsx]: Fixed

File: src/components/Tag/_tag.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
+.ssb-tag {
+  @include roboto;
+  @include reset-button;
+  align-items: center;
+  background: $ssb-white;
+  border: 1px solid $ssb-green-4;
+  border-radius: 21.5px;
```
Comment (kristofferlund) [inline src/components/Tag/_tag.scss]: The sketches are not right here, can we change this to 20px?

Comment (ssb-cgn) [reply src/components/Tag/_tag.scss]: fixed

**Pull request merged**